TRANSLATING.md

```diff
@@ -135,6 +141,22 @@ If you want to take your reviews to the next level, here are some more questions
 
 ## Translation Tips
 
+### How and How Much to adapt your translation
```

Comment:
I got some ideas for the title, feel free to dismiss

```suggestion
Adapting English to other languages
```
lower case title

```suggestion
### How and how much to adapt your translation
```

Reply:
Thanks @kevinzunigacuellar, I'll let @Yan-Thomas decide! Both would be fine to me. I suppose consider too whatever you think might be best suited to perhaps non-native English speakers. The "how and how much" is a nice English phrase, but only if it's widely understood/not confusing.

Reply:
I am happy with both 🎃